<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="TodosBackend Test Suite" tests="15" assertions="7" failures="8" errors="1" time="6.182188">
    <testsuite name="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" tests="13" assertions="7" failures="6" errors="1" time="5.015817">
      <testcase name="testRetrieveAllTasks" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="97" assertions="0" time="0.481315">
        <error type="PHPUnit_Framework_ExceptionWrapper">TasksApiTest::testRetrieveAllTasks
Error: Call to a member function json() on null

/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php:104
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:185
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:115
</error>
      </testcase>
      <testcase name="testRetrieveOneTask" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="125" assertions="0" time="0.318978">
        <failure type="PHPUnit_Framework_AssertionFailedError">TasksApiTest::testRetrieveOneTask
Invalid JSON was returned from the route. Perhaps an exception was thrown?

/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:426
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:355
/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php:133
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:185
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:115
</failure>
      </testcase>
      <testcase name="testCreateNewTask" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="150" assertions="3" time="0.343145"/>
      <testcase name="testUpdateExistingTask" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="168" assertions="0" time="0.377638">
        <failure type="PHPUnit_Framework_AssertionFailedError">TasksApiTest::testUpdateExistingTask
Invalid JSON was returned from the route. Perhaps an exception was thrown?

/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:426
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:298
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:324
/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php:175
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:185
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:115
</failure>
      </testcase>
      <testcase name="testDeleteExistingTask" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="186" assertions="0" time="0.377206">
        <failure type="PHPUnit_Framework_AssertionFailedError">TasksApiTest::testDeleteExistingTask
Invalid JSON was returned from the route. Perhaps an exception was thrown?

/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:426
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:298
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:324
/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php:191
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:185
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:115
</failure>
      </testcase>
      <testcase name="Warning" assertions="0" time="0.006404"/>
      <testcase name="testGetNotExistingTask" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="215" assertions="0" time="0.390151">
        <failure type="PHPUnit_Framework_AssertionFailedError">TasksApiTest::testGetNotExistingTask
Invalid JSON was returned from the route. Perhaps an exception was thrown?

/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:426
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:298
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:324
/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php:204
/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php:217
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:185
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:115
</failure>
      </testcase>
      <testcase name="testUpdateNotExistingTask" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="226" assertions="0" time="0.418056">
        <failure type="PHPUnit_Framework_AssertionFailedError">TasksApiTest::testUpdateNotExistingTask
Invalid JSON was returned from the route. Perhaps an exception was thrown?

/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:426
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:298
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:324
/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php:204
/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php:228
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:185
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:115
</failure>
      </testcase>
      <testcase name="testDeleteNotExistingTask" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="237" assertions="0" time="0.477245">
        <failure type="PHPUnit_Framework_AssertionFailedError">TasksApiTest::testDeleteNotExistingTask
Invalid JSON was returned from the route. Perhaps an exception was thrown?

/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:426
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:298
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:324
/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php:204
/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php:239
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:185
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:115
</failure>
      </testcase>
      <testcase name="testPagination" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="246" assertions="1" time="0.411212"/>
      <testcase name="testNameIsRequiredAndDefaultValues" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="258" assertions="1" time="0.441422"/>
      <testcase name="testPriorityHasToBeAnInteger" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="267" assertions="1" time="0.469133"/>
      <testcase name="testDoneHasToBeBoolean" class="TasksApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TaskApiTest.php" line="276" assertions="1" time="0.503912"/>
    </testsuite>
    <testsuite name="TaskApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TestApiTest.php" tests="2" assertions="0" failures="2" errors="0" time="1.166371">
      <testcase name="testShowAllTasks" class="TaskApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TestApiTest.php" line="14" assertions="0" time="0.593379">
        <failure type="PHPUnit_Framework_AssertionFailedError">TaskApiTest::testShowAllTasks
Invalid JSON was returned from the route. Perhaps an exception was thrown?

/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:426
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:355
/home/pedro/Code/Todos/TodosBackend/tests/TestApiTest.php:26
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:185
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:115
</failure>
      </testcase>
      <testcase name="testShowOneTask" class="TaskApiTest" file="/home/pedro/Code/Todos/TodosBackend/tests/TestApiTest.php" line="37" assertions="0" time="0.572992">
        <failure type="PHPUnit_Framework_AssertionFailedError">TaskApiTest::testShowOneTask
Invalid JSON was returned from the route. Perhaps an exception was thrown?

/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:426
/home/pedro/Code/Todos/TodosBackend/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php:355
/home/pedro/Code/Todos/TodosBackend/tests/TestApiTest.php:43
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:185
/home/pedro/.composer/vendor/phpunit/phpunit/src/TextUI/Command.php:115
</failure>
      </testcase>
    </testsuite>
  </testsuite>
</testsuites>
